docs/user-guide/geoprocessing-tool.md

```diff
@@ -0,0 +1,74 @@
+# GeoProcessing Tool
+
+This tool aims to provide a set of geo-processing utilities on layers present in map. WPS calls are made by the tool to produce the result to be displayed in the map. Supported WPS processes are *geo:buffer* and *gs:IntersectionFeatureCollection* (the [WPS plugin](https://docs.geoserver.org/latest/en/user/services/wps/install.html) need to be installed for your GeoServer version in order to use this tool).
+
+!!! note
+    The **GeoProcessing Tool** is not included by default in the MapStore configuration but it can be configured within an [application contexts](application-context.md#configure-plugins) if needed.
```

Comment:
```suggestion
    The **GeoProcessing Tool** is not included by default in the MapStore configuration but it can be configured within an [application contexts](application-context.md#configure-plugins) if needed or directly included in the standard [application configuration](../../developer-guide/local-config/).
```

Reply:
@ElenaGallo please check if the application configuration relative link is correct.